js/ui/tag_box.js

```diff
@@ -921,8 +921,13 @@ var TagBox = SelectBox.inherit({
     },
 
     _renderTag: function(item, $input) {
-        var value = commonUtils.ensureDefined(this._valueGetter(item), item),
-            $tag = this._getTag(value);
+        var value = this._valueGetter(item);
```

Comment:
I think that these changes contain potential BC: please check [this sample](https://codepen.io/anon/pen/jYOwjK) with your changes

Reply:
This sample works